assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -48,7 +48,7 @@ $filter-btn: '.esprs-button.ant-btn.ee-filter-bar__btn';
 			margin: 0;
 			&:focus,
 			&:hover {
-				border: 1px solid var(--ee-color-primary-low-contrast);
+				border: 2px solid var(--ee-color-primary-low-contrast);
```

Comment:
This change was needed because of this issue:

![Peek 2020-03-25 14-32](https://user-images.githubusercontent.com/1477453/77536804-9f549080-6ea5-11ea-822e-05a1423d61bc.gif)

Reply:
ok... might have to apply that change in a bunch of other places too